.github/actions/create-conda-envs/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
         # get cylc version...
         if [[ -z '${{ inputs.cylc_flow_version }}' ]]; then
           # ... from cylc library
-          CYLC_RELEASE="$(python ./docs/src/lib/cylc_release.py)"
+          CYLC_RELEASE="$(python ./src/lib/cylc_release.py)"
```

Comment:
This is a revert of #410

I think the proper fix will be to add an input to this action called something like `working_directory` that gives the path to `src`, and use that in this line